### ISSUE TITLE
finding tilt axis angle in miss-alignment / warp project

### ISSUE THREAD

**Issue by McHaillet** [warpem/miss-alignment#58]
Some code to calculate the tilt-axis angle via minimizing the difference between common lines. Its not producing good results (yet):

```python
In [2]: from warpylib import TiltSeries
   ...: from pathlib import Path
   ...: from torch_fourier_shift import fourier_shift_image_2d
   ...: from torch_refine_tilt_axis_angle import refine_tilt_axis_angle
   ...: import mrcfile
   ...: import torch
   ...: ts = Path('./').glob('*.xml')
   ...: alist = []
   ...: for x in ts:
   ...:     y = TiltSeries(x)
   ...:     with mrcfile.open(y.tilt_stack_path) as mrc:
   ...:         stack = torch.tensor(mrc.data).to('cuda:3')
   ...:     shifts = torch.stack((y.tilt_axis_offset_y, y.tilt_axis_offset_x), dim=1) / 10
   ...:     shifted = fourier_shift_image_2d(stack, -shifts.to('cuda:3'))
   ...:     axis_angle = refine_tilt_axis_angle(shifted[..., 18:-18], -90.6)
   ...:     print(axis_angle)
   ...:     alist.append(axis_angle)
   ...: print(f' mean axis angle = {sum(alist) / len(alist)}')
-80.6021392662072
-80.60269632911152
-100.59786073379279
-80.60261310154529
-90.00035000313541
-100.59786073379279
-100.59786073379279
-90.00162284656932
-80.6021392662072
-90.00150641586183
-89.99882130345983
-80.60132213922704
-100.59786073379279
```

**Comment by McHaillet** [comment]
Let's just rely on etomo patch tracking for estimating the tilt axis angle (or people should calibrate it better). I wrote a small gist on doing this with the WarpTools wrapper: https://gist.github.com/McHaillet/74596b3bea760001fd253de933baafe6
